### PR TITLE
[com_templates] Support plugin layout overrides

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue #22245 .

### Summary of Changes

This adds support for plugin layout overrides in Template Manager.

### Testing Instructions

Apply patch.
Go to template details. Switch to `Create Overrides` tab.
Check that plugins appear.
Check that overrides are created correctly.

### Documentation Changes Required
IDK.

### CODE REVIEW

**Review by brianteeman — APPROVED** [review]
approve new string

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on fcdef10ca68b57c582a75b95d89cadc3c1b6cc69<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22248">issues.joomla.org/tracker/joomla-cms/22248</a>.</sub>

File: administrator/language/en-GB/en-GB.com_templates.ini [hunk 1]
```diff
@@ -195,6 +195,7 @@ COM_TEMPLATES_OVERRIDE_SUCCESS="Successfully created the override."
 COM_TEMPLATES_OVERRIDES_COMPONENTS="Components"
 COM_TEMPLATES_OVERRIDES_MODULES="Modules"
 COM_TEMPLATES_OVERRIDES_LAYOUTS="Layouts"
+COM_TEMPLATES_OVERRIDES_PLUGINS="Plugins"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_templates.ini]: Please sort lines 196-197.

**Comment by laoneo** [comment]
I have tested this item :white_check_mark: successfully on fcdef10ca68b57c582a75b95d89cadc3c1b6cc69<br /><br />I was waiting for this since 3.7. Now it is much simpler to override the custom fields layouts. Didn't think it will be such a small pr to achieve it. Well done!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22248">issues.joomla.org/tracker/joomla-cms/22248</a>.</sub>

**Comment by ciar4n** [comment]
Thank you @SharkyKZ . Nice job.

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

Note: At Issue Tracker no Test is counted.

**Pull request merged**